css/CSS2/backgrounds/background-root-012.xht

```diff
@@ -7,7 +7,7 @@
   <link rel="help" href="http://www.w3.org/TR/CSS21/colors.html#background"/>
   <link rel="match" href="background-root-012-ref.xht" />
 
-  <meta name="flags" content="image interact"/>
```

Comment:
(not my spec) This test seems to have two parts. one that matches the ref, and one where the user is invited to resize the window and observe effects. Only the first part is non interactive. I don't believe that removing the flag without modifying the test of splitting it in two would be correct.